lib/system/generic/xlnx/sdt.h

```diff
@@ -0,0 +1,27 @@
+/*
+ * Copyright (c) 2022-2023 Advanced Micro Devices, Inc. All Rights Reserved.
+ *
+ * SPDX-License-Identifier: BSD-3-Clause
+ */
+
+/*
+ * @file	generic/xlnx/sdt.h
+ * @brief	Xilinx-AMD Specific symbols for System Device Tree Flow
+ */
+
+#ifndef __METAL_GENERIC_SYS_XLNX_SDT_H__
+#define __METAL_GENERIC_SYS_XLNX_SDT_H__
+
+/* The Xilinx-AMD System Device Tree (SDT) Flow BSP provides this file. */
+#ifdef SDT
```

Comment:
needed? you include this file only if `SDT` is defined

Reply:
correct - this patch includes the bspconfig.h and provides symbols if system device tree bsp is included. 
we iner this based on the symbol SDT being present.